local-dev/templates/rawls.conf

```diff
@@ -93,8 +93,8 @@ dataRepo {
 
 resourceBuffer {
   projectPool {
-    regular = "cwb_ws_dev_v7"
-    exfiltrationControlled = "vpc_sc_v10"
+    regular = "cwb_ws_dev_v8"
+    exfiltrationControlled = "vpc_sc_v13"
```

Comment:
Whoops, should have updated these when I switched to the new pools but I didn't know they existed!